Create inventory, group, and hosts in Tower from static 'web' inventory -- issue #136

kyleabenson opened the pull request:
#136 
This will create an inventory, group, and hosts in Tower from the statically defined workshop 'web' group.

Currently, it does not associate the hosts with the group as the Ansibler_Tower modules (and tower_cli) do not allow a group option to be passed when creating hosts.

I will add something in the future, likely via URI, to make a update the group for each created host

tima commented:
Thanks for this @kyleabenson. 

How about some docs explaining it's purpose and how to use it?

kyleabenson commented:
@tima Good call -- I moved this into a folder to match the aws_lab_setup style, included a README and some relevant images. I also made an optional reference in the workshop docs for tower_basic_setup, and cleaned up a few typos there as well as in the solutions guide for that workshop.

tima commented:
Thanks @kyleabenson.
